src/components/templates/TalkShowStudio/components/Player/Player.tsx

```diff
@@ -66,18 +66,19 @@ const Player: React.FC<VideoPlayerProps> = ({
     };
   }, [audioTrack]);
 
-  const IconContainerClasses = (isOn: boolean) =>
+  const iconContainerClasses = (isOff: boolean) =>
     classNames("Player__icon", {
-      "Player__icon--on": isOn,
+      "Player__icon--off": isOff,
     });
 
   return (
     <div ref={container} className={classNames("Player", containerClass)}>
+      <div className="Player__background" />
       {showButtons && (
         <div className="Player__overlay">
           <div className="Player__buttons">
             <div
-              className={IconContainerClasses(!isMicOn)}
+              className={iconContainerClasses(!isMicOn)}
```

Comment:
We shouldn't have to add the `!` here. The helper should handle any negations it may need inside itself. The calling code shouldn't have to care.